templates/base.html

```diff
@@ -49,7 +52,9 @@
         <link rel="stylesheet" href="{% static 'django_colortag.css' %}" />
         <script src="{% static 'js/polyfill-find.js' %}"></script>
         <script src="{% static 'js/focus-visible-polyfill.min.js' %}"></script>
-        <script src="{% static 'js/aplus.js' %}"></script>
+        <script 
+            src="{% static 'js/aplus.js' %}"
+            course-code="{{ course.code }}"></script>
```

Comment:
This could be a separate `meta` tag containing the course code. Would also remove the IE11 hack.